cli/standalone/binary.rs

```diff
@@ -575,11 +575,29 @@ impl<'a> DenoCompileBinaryWriter<'a> {
       None => None,
     };
     let mut vfs = VfsBuilder::new();
+    let mut remote_modules_size = 0.0;
     let npm_snapshot = match self.npm_resolver.as_inner() {
       InnerCliNpmResolverRef::Managed(managed) => {
         let snapshot =
           managed.serialized_valid_snapshot_for_system(&self.npm_system_info);
         if !snapshot.as_serialized().packages.is_empty() {
+          let npm_snapshot = managed.snapshot();
+          // make sure that graph.npm_packages.is_empty() isn't empty
+          if !graph.npm_packages.is_empty() {
+            for module in graph.modules() {
+              if let deno_graph::Module::Npm(module) = module {
+                let nv = module.nv_reference.nv();
+                if let Ok(package) =
+                  npm_snapshot.resolve_package_from_deno_module(nv)
+                {
+                  // package.id.clone() is found now
+                  if let Ok(size) = managed.package_size(&package.id) {
+                    remote_modules_size += size as f64;
+                  }
+                }
+              }
+            }
+          }
```

Comment:
We shouldn't output this because npm packages are stored in the file system whose size is already displayed.

Let's wait for https://github.com/denoland/deno/pull/27395 to land because implementing this after that will basically just be getting the size from the `RemoteModulesStoreBuilder`.

Reply:
This PR landed.